api/src/main/resources/moduleApplicationContext.xml

```diff
@@ -55,4 +55,8 @@
             </list>
         </property>
     </bean>
+
+  <bean name="patientCalculationService"

```

Comment:
this doesn't seem right, i am assuming you added this just for the sake of the test. this should be causing bean duplication with integration since calculation module adds the same

Reply:
declared the bean here because it could not Autowired to `AbstractPatientCalculation`, but since we are going to stick with `Context.get` I'll revert it.